plugins/woocommerce/tests/e2e-pw/utils/editor.js

```diff
@@ -53,11 +53,34 @@ const fillPageTitle = async ( page, title ) => {
 	await ( await getCanvas( page ) ).getByLabel( 'Add title' ).fill( title );
 };
 
-const insertBlock = async ( page, blockName ) => {
-	await page.getByLabel( 'Toggle block inserter' ).click();
+const insertBlock = async ( page, blockName, wpVersion = null ) => {
+	await page
+		.getByRole( 'button', {
+			name: 'Toggle block inserter',
+			expanded: false,
+		} )
+		.click();
 	await page.getByPlaceholder( 'Search', { exact: true } ).fill( blockName );
 	await page.getByRole( 'option', { name: blockName, exact: true } ).click();
-	await page.getByLabel( 'Toggle block inserter' ).click();
+
+	// In WP 6.6 'Toggle block inserter' button closes the inserter as expected,
+	// but trying to immediately open it again will fail in Playwright, while manually it works.
+	// We have tests that insert multiple blocks and fail because of this.
+	// Using the new 'Close block inserter' button added in WP 6.6 works fine.
+	if ( wpVersion && wpVersion <= 6.5 ) {
+		await page
```

Comment:
Unfortunately this won't work, because both Close and Toggle buttons are present and Playwright will throw an error here. In 6.5 the Close is hidden (??) and in 6.6 clicking Toggle will break the next action (hence this fix).